addOns/accessControl/src/main/javahelp/org/zaproxy/zap/extension/accessControl/resources/help/contents/concepts.html

```diff
@@ -69,6 +69,32 @@ <h3>Testing procedure</h3>
 </ul>
 <strong>Note:</strong> Access control testing is not allowed in <code>Safe</code> mode nor <code>Protected</code> if the context is not in scope.
 
+<H2>Latest Code</H2> 
+
+<a href="https://github.com/zaproxy/zap-extensions/tree/main/addOns/accessControl">accessControl</a>
```

Comment:
Don't mind much the general one, for other scan rules we link directly to the scan rules' class.

Reply:
There is `AccessControlScannerThread` (that it does not extend the base scan rules does not make the code less relevant to the raised alerts).